apps/payment/models.py

```diff
@@ -1,15 +1,17 @@
 from django.db import models
-
+from datetime import datetime
 from apps.authentication.models import CustomUser
 from apps.payment.enums import MemberType
+from dateutil.relativedelta import relativedelta
 
 class Credit(models.Model):
     value = models.IntegerField(blank=False, null=False)
     creation_date = models.DateField(auto_now_add=True, blank=False, null=False)
-    user = models.ForeignKey(CustomUser, on_delete=models.CASCADE, blank=False, null=False)
+    user = models.OneToOneField(CustomUser, on_delete=models.CASCADE, blank=False, null=False)
```

Comment:
¿Porqué el cambio de ForeignKey a OneToOne?
El el UML se puede ver que es one to many. https://drive.google.com/file/d/1RUGeWm69W2DxBIdpvVHHlDddUT2H9cu4/view
En el caso del Memebership un usuario debe tener su historial de membresias por lo que se debe guardar todas, por eso se indica la fecha de fin e inicio.
Por otro lado, con credits estoy dudando, se le puede hacer varias interpretaciones.